fe/fe-core/src/main/java/org/apache/doris/datasource/property/storage/HdfsCompatibleProperties.java

```diff
@@ -30,14 +35,16 @@ protected HdfsCompatibleProperties(Type type, Map<String, String> origProps) {
         super(type, origProps);
     }
 
-    protected Configuration configuration;
+    @Getter
+    protected HadoopAuthenticator hadoopAuthenticator = new HadoopSimpleAuthenticator(new SimpleAuthenticationConfig());
 
     @Override
     protected String getResourceConfigPropName() {
         return "hadoop.config.resources";
     }
 
     public Configuration getHadoopConfiguration() {
```

Comment:
This method can be removed, the `hadoopStorageConfig` already annotated with @Getter